FE 3.1.0-rc02 can't start

<!-- (At least include the following, feel free to add if you have more content) -->

### Steps to reproduce the behavior (Required)
```
docker run -it --rm 172.26.92.142:5000/starrocks/allin1-ubi:3.1.0-rc02 bash

cd /data/deploy/starrocks

./bin/start_fe.sh

```

fe won't start. check `fe.log`
```
2023-07-24 03:09:07,431 INFO (main|1) [StarRocksFE.start():127] StarRocks FE starting, version: 3.1.0-rc02-fb4c4ee
2023-07-24 03:09:07,438 INFO (main|1) [FrontendOptions.initAddrUseIp():249] Use IP init local addr, IP: /172.17.0.5
2023-07-24 03:09:07,712 INFO (main|1) [Auth.grantRoleInternal():837] grant operator to 'root'@'%', isReplay = true
2023-07-24 03:09:07,740 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():286] create built-in role root[-1]
2023-07-24 03:09:07,744 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():286] create built-in role db_admin[-2]
2023-07-24 03:09:07,745 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():286] create built-in role cluster_admin[-3]
2023-07-24 03:09:07,745 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():286] create built-in role user_admin[-4]
2023-07-24 03:09:07,745 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():286] create built-in role public[-5]
2023-07-24 03:09:07,745 INFO (main|1) [GlobalStateMgr.initAuth():1101] using new privilege framework..
2023-07-24 03:09:07,940 INFO (main|1) [NodeMgr.getHelperNodes():647] get helper nodes: [172.17.0.5:9010]
2023-07-24 03:09:07,949 ERROR (main|1) [StarRocksFE.start():190] StarRocksFE start failed
java.lang.NoClassDefFoundError: org/slf4j/spi/LoggingEventBuilder
        at org.apache.logging.slf4j.Log4jLoggerFactory.newLogger(Log4jLoggerFactory.java:50) ~[log4j-slf4j2-impl-2.19.0.jar:2.19.0]
        at org.apache.logging.slf4j.Log4jLoggerFactory.newLogger(Log4jLoggerFactory.java:33) ~[log4j-slf4j2-impl-2.19.0.jar:2.19.0]
        at org.apache.logging.log4j.spi.AbstractLoggerAdapter.getLogger(AbstractLoggerAdapter.java:53) ~[log4j-api-2.19.0.jar:2.19.0]
        at org.apache.logging.slf4j.Log4jLoggerFactory.getLogger(Log4jLoggerFactory.java:33) ~[log4j-slf4j2-impl-2.19.0.jar:2.19.0]
        at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:363) ~[slf4j-api-1.7.30.jar:1.7.30]
        at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:388) ~[slf4j-api-1.7.30.jar:1.7.30]
        at com.starrocks.plugin.PluginInfo.<clinit>(PluginInfo.java:61) ~[starrocks-fe.jar:?]
        at com.starrocks.qe.AuditLogBuilder.<init>(AuditLogBuilder.java:61) ~[starrocks-fe.jar:?]
        at com.starrocks.plugin.PluginMgr.initBuiltinPlugins(PluginMgr.java:123) ~[starrocks-fe.jar:?]
        at com.starrocks.plugin.PluginMgr.init(PluginMgr.java:100) ~[starrocks-fe.jar:?]
        at com.starrocks.server.GlobalStateMgr.initialize(GlobalStateMgr.java:1067) ~[starrocks-fe.jar:?]
        at com.starrocks.StarRocksFE.start(StarRocksFE.java:133) ~[starrocks-fe.jar:?]
        at com.starrocks.StarRocksFE.main(StarRocksFE.java:84) ~[starrocks-fe.jar:?]
Caused by: java.lang.ClassNotFoundException: org.slf4j.spi.LoggingEventBuilder
        at java.net.URLClassLoader.findClass(URLClassLoader.java:387) ~[?:1.8.0_382]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[?:1.8.0_382]
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352) ~[?:1.8.0_382]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_382]
        ... 13 more
```


### Expected behavior (Required)

FE can start with no issue

### Real behavior (Required)

FE won't start.

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`

```
[root@094fd46cd6bf fe]# ./bin/show_fe_version.sh 
Commit hash: fb4c4ee
Build type: RELEASE
Build time: 2023-07-22 14:20:43
Build user: StarRocks@localhost
Java compile version: openjdk full version "1.8.0_362-b08"
```

+1
faced the same problem:
```bash
2023-07-24 06:13:52,199 ERROR (main|1) [MetaHelper.checkMetaDir():162] meta dir /home/ubuntu/Projects/starrocks/StarRocks-3.1.0-rc02/fe/meta does not exist
2023-07-24 06:13:52,203 ERROR (main|1) [StarRocksFE.start():190] StarRocksFE start failed
com.starrocks.common.InvalidMetaDirException: null
	at com.starrocks.leader.MetaHelper.checkMetaDir(MetaHelper.java:163) ~[starrocks-fe.jar:?]
	at com.starrocks.StarRocksFE.start(StarRocksFE.java:125) ~[starrocks-fe.jar:?]
	at com.starrocks.StarRocksFE.main(StarRocksFE.java:84) ~[starrocks-fe.jar:?]
2023-07-24 06:14:17,166 INFO (main|1) [StarRocksFE.start():127] StarRocks FE starting, version: 3.1.0-rc02-fb4c4ee
2023-07-24 06:14:17,172 INFO (main|1) [FrontendOptions.initAddrUseIp():249] Use IP init local addr, IP: /172.17.0.1
2023-07-24 06:14:17,434 INFO (main|1) [Auth.grantRoleInternal():837] grant operator to 'root'@'%', isReplay = true
2023-07-24 06:14:17,460 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():286] create built-in role root[-1]
2023-07-24 06:14:17,465 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():286] create built-in role db_admin[-2]
2023-07-24 06:14:17,465 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():286] create built-in role cluster_admin[-3]
2023-07-24 06:14:17,465 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():286] create built-in role user_admin[-4]
2023-07-24 06:14:17,466 INFO (main|1) [AuthorizationMgr.initBuiltinRoleUnlocked():286] create built-in role public[-5]
2023-07-24 06:14:17,466 INFO (main|1) [GlobalStateMgr.initAuth():1101] using new privilege framework..
2023-07-24 06:14:17,643 INFO (main|1) [NodeMgr.getHelperNodes():647] get helper nodes: [172.17.0.1:9010]
2023-07-24 06:14:17,650 ERROR (main|1) [StarRocksFE.start():190] StarRocksFE start failed
java.lang.NoClassDefFoundError: org/slf4j/spi/LoggingEventBuilder
	at org.apache.logging.slf4j.Log4jLoggerFactory.newLogger(Log4jLoggerFactory.java:50) ~[log4j-slf4j2-impl-2.19.0.jar:2.19.0]
	at org.apache.logging.slf4j.Log4jLoggerFactory.newLogger(Log4jLoggerFactory.java:33) ~[log4j-slf4j2-impl-2.19.0.jar:2.19.0]
	at org.apache.logging.log4j.spi.AbstractLoggerAdapter.getLogger(AbstractLoggerAdapter.java:53) ~[log4j-api-2.19.0.jar:2.19.0]
	at org.apache.logging.slf4j.Log4jLoggerFactory.getLogger(Log4jLoggerFactory.java:33) ~[log4j-slf4j2-impl-2.19.0.jar:2.19.0]
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:363) ~[slf4j-api-1.7.30.jar:1.7.30]
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:388) ~[slf4j-api-1.7.30.jar:1.7.30]
	at com.starrocks.plugin.PluginInfo.<clinit>(PluginInfo.java:61) ~[starrocks-fe.jar:?]
	at com.starrocks.qe.AuditLogBuilder.<init>(AuditLogBuilder.java:61) ~[starrocks-fe.jar:?]
	at com.starrocks.plugin.PluginMgr.initBuiltinPlugins(PluginMgr.java:123) ~[starrocks-fe.jar:?]
	at com.starrocks.plugin.PluginMgr.init(PluginMgr.java:100) ~[starrocks-fe.jar:?]
	at com.starrocks.server.GlobalStateMgr.initialize(GlobalStateMgr.java:1067) ~[starrocks-fe.jar:?]
	at com.starrocks.StarRocksFE.start(StarRocksFE.java:133) ~[starrocks-fe.jar:?]
	at com.starrocks.StarRocksFE.main(StarRocksFE.java:84) ~[starrocks-fe.jar:?]
Caused by: java.lang.ClassNotFoundException: org.slf4j.spi.LoggingEventBuilder
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581) ~[?:?]
	at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
	... 13 more

```

--- a/fe/fe-core/pom.xml
+++ b/fe/fe-core/pom.xml
@@ -350,12 +350,6 @@ under the License.
             <artifactId>log4j-slf4j-impl</artifactId>
         </dependency>
 
-        <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-slf4j2-impl -->
-        <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-slf4j2-impl</artifactId>
-        </dependency>
-
         <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-1.2-api -->
         <dependency>
             <groupId>org.apache.logging.log4j</groupId>

--- a/fe/pom.xml
+++ b/fe/pom.xml
@@ -374,13 +374,6 @@ under the License.
                 <version>${log4j.version}</version>
             </dependency>
 
-            <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-slf4j2-impl -->
-            <dependency>
-                <groupId>org.apache.logging.log4j</groupId>
-                <artifactId>log4j-slf4j2-impl</artifactId>
-                <version>${log4j.version}</version>
-            </dependency>
-
             <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-1.2-api -->
             <dependency>
                 <groupId>org.apache.logging.log4j</groupId>
@@ -484,8 +477,12 @@ under the License.
                         <artifactId>slf4j-log4j12</artifactId>
                     </exclusion>
                     <exclusion>
-                        <artifactId>zookeeper</artifactId>
                         <groupId>org.apache.zookeeper</groupId>
+                        <artifactId>zookeeper</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.logging.log4j</groupId>
+                        <artifactId>log4j-slf4j2-impl</artifactId>
                     </exclusion>
                 </exclusions>
             </dependency>

--- a/fe/spark-dpp/pom.xml
+++ b/fe/spark-dpp/pom.xml
@@ -164,12 +164,6 @@ under the License.
             <artifactId>log4j-slf4j-impl</artifactId>
             <scope>provided</scope>
         </dependency>
-
-        <dependency>
-            <groupId>org.apache.logging.log4j</groupId>
-            <artifactId>log4j-slf4j2-impl</artifactId>
-            <scope>provided</scope>
-        </dependency>
     </dependencies>
 
     <build>

